src/naive/datetime/mod.rs

```diff
@@ -722,6 +722,22 @@ impl NaiveDateTime {
     pub fn format<'a>(&self, fmt: &'a str) -> DelayedFormat<StrftimeItems<'a>> {
         self.format_with_items(StrftimeItems::new(fmt))
     }
+
+    /// Converts the `NaiveDateTime` into the timezone-aware `DateTime<Tz>`
+    /// with the provided timezone, if possible.
+    ///
+    /// This is experimental and might be removed in the future. Feel free to
```

Comment:
Why are we calling this experimental again?

Reply:
It was proposed in this comment. Apparently this API gets reinvented a lot. https://github.com/chronotope/chrono/pull/711#issuecomment-1157647197

Reply:
I don't think labeling this as experimental isn't helpful because it's not clear what downstream users should do with that information.